task/net/src/bsp/medusa_a.rs

```diff
@@ -0,0 +1,138 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! BSP for the Sidecar rev B
+//!
+//! Right now, this is identical to the rev A BSP, but that may change in the
+//! future, so they're kept separate.
```

Comment:
looks like this was copied and pasted and should, instead, say:
```suggestion
//! BSP for Medusa rev A
```